### PR TITLE
docs: update pull request template

### PR DESCRIPTION
**Pull request by fhennig**

### CODE REVIEW

**Review by razvan — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -14,4 +14,19 @@
...
+- [ ] Integration tests ran successfully
 ```
+
+<details>
+<summary>TIP: Running integration tests with a new product image</summary>
+
+The image can be built and uploaded to the kind cluster with the following commands:
+
+```shell
+./build_product_images.py --product <product> --image_version <stackable-image-version>
```
Comment (lfrancke) [inline .github/pull_request_template.md:27]: This file does not exist

**Review by lfrancke — COMMENTED** [review]
I  know it's already been committed but I do have a change request...

File: .github/pull_request_template.md (hunk 1 above)
Comment (razvan) [reply .github/pull_request_template.md:27]: https://github.com/stackabletech/docker-images/pull/610

Comment (fhennig) [reply .github/pull_request_template.md:27]: thanks both of you, sorry about that!

**Pull request merged**